src/ocean/task/util/StreamProcessor.d

```diff
@@ -184,6 +184,7 @@ class StreamProcessor ( TaskT : Task )
 
         if (throttler_config.resume_point == size_t.max)
             throttler_config.resume_point = total / 5;
+		else
```

Comment:
@gkotian I'm not sure this is how you want to do this. Because you want to actually execute first `enforce` only if the `suspend_point != size_t` max. If you remove `else`, both of the enforces are becoming redundant, but not equivalent the original code (or code that was supposed to be here) - only test if the `suspend point`/`resume_point` is not `size_t`.